### PR TITLE
Added the Translate to POS function. 

### PR DESCRIPTION
**Pull request by diptanil-roy**

### CODE REVIEW

**Review by kdettman — CHANGES_REQUESTED** [review]
On a quick review this looks pretty good. I'll check the branch out to play with it a little. Otherwise, can you change all the comments that aren't the initial docstring to just `#`? Its a little hard to read. At first I thought the method was once big docstring.

**Comment by diptanil-roy** [comment]
> On a quick review this looks pretty good. I'll check the branch out to play with it a little. Otherwise, can you change all the comments that aren't the initial docstring to just `#`? Its a little hard to read. At first I thought the method was once big docstring.

Addressed in the latest commit.

**Comment by emapple** [comment]
In the future, can you please create a new branch for every pull request? At first I saw 26 commits and thought this was a huge overhaul

**Comment by diptanil-roy** [comment]
> In the future, can you please create a new branch for every pull request? At first I saw 26 commits and thought this was a huge overhaul

Sure thing. I was working in this branch locally, so just kept pushing it. But yeah, will do.

**Review by kdettman — CHANGES_REQUESTED** [review]
The entire method `translate_to_pos` is incorrectly indented. Please indent properly.

**Comment by diptanil-roy** [comment]
> The entire method `translate_to_pos` is incorrectly indented. Please indent properly.

Done. Apologies for that.

File: src/book.py [hunk 1]
```diff
@@ -309,6 +308,49 @@ def snippet(self, length, on, groups=1, non_cont=False, with_replacement=True,
             self._text = snip._text
         else:
             return snip
+        
+    def translate_to_pos(self:BookText, inplace=False):
```
Comment (kdettman) [inline src/book.py]: I got an error when `self` was set to be a `BookText` object. I don't think `self` being set is really necessary and it throws an error so just change`self:BookText` to `self`.

**Review by kdettman — CHANGES_REQUESTED** [review]

File: src/book.py (hunk 1 above)
Comment (diptanil-roy) [reply src/book.py]: Fixed.

**Comment by kdettman** [comment]
Excellent, I was just about to ask for that change as well.

**Comment by diptanil-roy** [comment]
> Excellent, I was just about to ask for that change as well.

Cool. If you are satisfied with it, please merge it.

**Review by kdettman — APPROVED** [review]

**Pull request merged**